packages/react/src/components/dateInput/components/datePicker/types/DayPickerProps.ts

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable import/no-cycle */
```

Comment:
I wonder if it would be good idea to rename this file DayPickerProps -> DatePickerProps?